src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -138,13 +138,20 @@
   @#   The domain name of the API remote host.
   @# @@param port [Integer]
   @#   The port on which to connect to the remote host.
+  # TODO(landrito): add documentation specifying that the channel updater_proc and keyfile
+  # must be used independently in relation to each other.
   @# @@param channel [Channel]
   @#   A Channel object through which to make calls.
-  @# @@param chan_creds [Grpc::ChannelCredentials]
```

Comment:
We should not remove this in case the user wants to set, e.g., non-default SSL creds. Could we have one parameter that accepts either `String` or `Grpc::ChannelCredentials`?